parsers/HOPS.py

```diff
@@ -29,21 +62,26 @@ def fetch_production(zone_key='HR', session=None, target_datetime=None,
     df = pd.DataFrame.from_dict(obj['resources']).set_index('sourceName')
 
     # Get the wind power generation
-    wind = df['value'].loc['Proizvodnja VE']
+    # In some cases value may be negative. It looks like an issue at data source side.
+    wind = abs(df['value'].loc['Proizvodnja VE'])
```

Comment:
maybe it makes more sense to consider negative values as 0? We could do max([0, ...])

Reply:
Converted this one to the positive value since noticed that HOPS UI do the same.
They have negative value in API response but when display it at website it gets converted to the positive value. It makes me think that there may be some issues in the data itself.

Reply:
Let's do it the same way as they do it - thanks for clarifying